### ISSUE TITLE
Feature Request: autostart server

### ISSUE THREAD

**Issue by stephio** [lmstudio-ai/.github#36]
First of all; Thank you for this great app!

Is there any plan in the near future to have a possibility to start a server from prompt? 
I think this would be a very useful feature.

**Comment by redet-G** [comment]
Yes, We have `lms cli` now. Check the documentation here https://lmstudio.ai/blog/lms
